database/operations/kv.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * This file is part of Moderent.
+ *
+ * Moderent is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Moderent is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Afferto General Public License
+ * along with Moderent.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { database } from "../database.ts";
+import { Collection } from "mongo";
+
+export interface Kv {
+  key: string;
+  // deno-lint-ignore no-explicit-any
+  value: any;
+}
```

Comment:
I think having support for generic type would be great.

```suggestion
// deno-lint-ignore no-explicit-any
export interface Kv<T = any> {
  key: string;
  value: T;
}
```

Reply:
Can you suggest how should the `get` and `set` functions be modified, too?